src/snippets/testcase.ts

```diff
@@ -0,0 +1,48 @@
+import { SnippetItem } from ".";
+
+export const testCaseMinWithHttp = <SnippetItem>{
+  label: "testcase: Minimal spec. with http request",
```

Comment:
```suggestion
  label: "Minimal spec. with http request",
```

Looks better IMO. Also applies for `http`.

Reply:
I understand you. However, for first few version it would be less convenient for untrained eyes. Let's keep it for now. Will remove later if reported. 🙏